packages/frontend-api/src/Resources/config/graphql-types/QueryDecorator.types.yaml

```diff
@@ -17,7 +17,7 @@ QueryDecorator:
                 description: "Returns list of ordered products that can be paginated using `first`, `last`, `before` and `after` keywords"
             product:
                 type: 'Product'
-                resolve: "@=resolver('product', [args['uuid']])"
+                resolve: "@=resolver('productByUuid', [args['uuid']])"
                 args:
                     uuid:
                         type: "Uuid!"
```

Comment:
This check will not let pass anything except UUID 😉

Reply:
yup, fixed in c16720fe9